roles/foreman_server_repositories/meta/main.yml

```diff
@@ -6,12 +6,12 @@ dependencies:
       - ansible_distribution_major_version == '7'
   - role: ansible_repositories
     when: foreman_server_repositories_ansible|bool
-  - role: puppet_repositories
+  - role: theforeman.operations.puppet_repositories
```

Comment:
the forklift `puppet_repositories` role would erase `puppet5-release` before installing `puppet6-release`, which the operations one doesn't.

we had that for p5 to p6 upgrades, as their RPM would not conflict but not obsolete the old one…

the p7 files seem to have the same issue, given we don't support p7 yet it's not an issue today, but will be in future

Reply:
Sounds like when we add p7 over there we should add that ability.

Reply:
Yeah, I was thinking something along the lines of:

```yaml
supported_versions: [6,7]
removed_versions: "{{ supported_versions | difference([puppet_version]) }}"
```

(and tasks doing so)

Reply:
Sounds like we should file an issue over on the collection to track this?

Reply:
https://github.com/theforeman/foreman-operations-collection/issues/20

Reply:
Thanks for filing it!

Reply:
Thanks for fixing it!